tests/conftest.py

```diff
@@ -1,7 +1,8 @@
 import yaml
 import pytest  # type: ignore
 
-from looker_sdk import client, methods, models
+from looker_sdk.sdk.api40 import methods, models
+import looker_sdk as client
```

Comment:
let's just import `looker_sdk` without aliasing now that `client` no longer exists and use that everywhere

Reply:
required to avoid a conflict in a method name